### PR TITLE
Implement website light theme design. Fix #11

### LINKED ISSUE #11: Implement intended visual designs for light theme

### PR DESCRIPTION
**Pull request by thibaudcolas**
Depends on #65. This implements the intended design for the light theme of the site (#11).  See [Figma](https://www.figma.com/file/v1WfquYwTNSkfKZp5Ny51Z/DjangoCon-Europe-2023-Edinburgh-website-design?node-id=138%3A170&t=U1fmu01YDjM6ydqX-4).

[Deploy preview link](https://deploy-preview-58--2023-djangocon-eu-previews.netlify.app/).

Screenshot:

deploy-preview-58--2023-djangocon-eu-previews netlify app_

Current progress – for this PR:

- [x] Header & logo size
- [x] Header sub-navigation 
- [x] Hero "polaroid" pictures
- [x] Homepage hero text
- [x] Homepage hero background
- [x] Footer city outline
- [x] Correct "GB map" background image
- [x] Cross-browser testing
- [x] Windows high contrast mode support
- [x] News section layout
- [x] Conference "about" section layout
- [x] Conference dates section layout
- [x] Dates section overlap at specific viewport widths
- [x] Code refactoring
- [x] Code self-review
- [x] Documentation where appropriate
- [x] Cross-browser testing
- [x] Licensing & sourcing info for visuals

## Implementation notes

The designs of the footer skyline, the group picture, and the Edinburgh "skyline at night" photos have all been acquired with their respective rightholders. Have added appropriate licensing information in the repository docs, and full files / order info in Google Drive.

The non-SVG visuals have been exported in heavily-compressed formats with [Squoosh](https://squoosh.app/). The original files are available in Google Drive.

### CODE REVIEW

**Comment by thibaudcolas** [comment]
This is almost ready to go – just need to clean up the code and commits, do more testing in Firefox, and update any documentation.

**Comment by thibaudcolas** [comment]
I have rebased this onto #65 which contains a few small accessibility styling fixes for components otherwise untouched in the redesign, and have also pushed a few commits straight to `main` for things that were clearly wrong and had no visual impact (alt text, broken links, commented out code).

File: _layouts/home.html [hunk 1]
```diff
@@ -113,30 +142,27 @@ <h3 class="card-title">Code of Conduct</h3>
...
-      <article class="news-title medium-4 column">
-        <div class="card {% if post.category == "Sponsorship" %}card-theme-brand-color3{% else %}card-theme-brand-color2{% endif %}">
-          <div class="card-section">
-            <span class="post-meta">{{ post.date | date: "%d %b %Y" }}</span>
-            <h3>
-              <a href="{{ post.url | prepend: site.baseurl }}">{{ post.title | escape }}</a>
-            </h3>
-          </div>
-        </div>
+      {% unless post.hidden or post.category == 'talk' or post.category == 'tutorial' %}
```
Comment (thibaudcolas) [inline _layouts/home.html:158]: This and below – copy of `post-list.html`, so we get the same results on the news page and here.

File: _src/scss/_settings.scss [hunk 2]
```diff
@@ -161,7 +164,7 @@ $block-grid-max: 8;
 // 4. Base Typography
 // ------------------
 
-$header-font-family: Metamorphous, -apple-system, BlinkMacSystemFont, 'Segoe UI', system-ui, Roboto, 'Helvetica Neue', Arial, sans-serif, Apple Color Emoji, 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
+$header-font-family: $body-font-family;
```
Comment (thibaudcolas) [inline _src/scss/_settings.scss:167]: We only want to use Metamorphous for the largest headings (h1, h2), so I’ve separately set those to use the correct font in `_global.scss`.

**Comment by thibaudcolas** [comment]
@sabderemane I think this is now ready for review 🤔 the PR does quite a lot in one go, which isn’t ideal, but I can’t think of a good way to split it further.

I’ve added a few code or PR comments for things that I thought could help with clarification, and created follow-up tasks as GitHub issues.

**Comment by sabderemane** [comment]
Amazing! Thanks @thibaudcolas, I'll check that 🤗

**Review by sabderemane — APPROVED** [review]
LGTM 🎉 Thank you for your work

**Pull request merged**